BackEnd/src/main/java/MusicAPI/harmonicsKB/SectionType.java

```diff
@@ -0,0 +1,14 @@
+package MusicAPI.harmonicsKB;
+
+/**
+ * Created by ben on 2/7/2016.
+ */
+public enum SectionType {
```

Comment:
All these objects should be in a separate folder probably to making understanding easier